packages/amplify-nodejs-function-template-provider/src/providers/graphqlRequestProvider.ts

```diff
@@ -4,30 +4,40 @@ import fs from 'fs-extra';
 import {
   AmplifySupportedService, exitOnNextTick,
 } from 'amplify-cli-core';
+import { $TSContext } from 'amplify-cli-core';
```

Comment:
This can be combined with the import above

Reply:
thank you, modified as suggested